src/components/searchResultsRow.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { getFriendlyRecordingRangeForHistory } from '../lib/utils';
+
+
+// Ported from explore.js
+function getPopulation(history) {
+  if (history) {
+    return history[0].optout ? 'release' : 'prerelease';
+  }
+  return '-';
+}
+
+const SearchResultsRow = ({
+    probeId,
+    rowData,
+    selectedChannel,
+    revisions,
+    channelInfo,
+    doExposeProbeDetails
+  }) => {
+
+  // TODO: possibly move this elsewhere.
+  let channelToUse = selectedChannel;
+  if (channelToUse === 'any') {
+    channelToUse = ['release', 'beta', 'nightly'].find(c => c in rowData.history);
+  }
+  const history = rowData.history[channelToUse];
+
+  // TODO: What happens with undefined history? Affects description and population.
+  return (
+    <tr onClick={() => doExposeProbeDetails(probeId, rowData)}>
+      <td className="search-results-field-"><span className="btn btn-outline-secondary btn-sm">+<span /></span></td>
```

Comment:
I would avoid the empty `<span />` if possible.

Reply:
An artifact of bootstrap/current styling that I didn't modify. I want to rework all the markup down the road - we also have inconsistent ids and classes (kebab-case vs snake_case etc).